docs/excel/run-code-on-document-open.md

```diff
@@ -73,9 +74,31 @@ async function onChange(event) {
     });
   });
 }
+```
+
+The following PowerPoint add-in code shows how to register an event handler for selection change events from the PowerPoint document. If you configure your add-in to load on document open, this code will register the event handler when the document is opened. You can handle change events before the task pane is opened.
+
+```JavaScript
+// This is called as soon as the document opens.
+// Put your startup code here.
+Office.onReady(info => {
+  if (info.host === Office.HostType.PowerPoint) {
+    Office.context.document.addHandlerAsync(Office.EventType.DocumentSelectionChanged, onChange);
+    console.log("A handler has been registered for the onChanged event.");
+  }
+});
 
+/**
+ * Handle the changed event from the PowerPoint document.
+ *
+ * @param event The event information from PowerPoint
+ */
+async function onChange(event) {
+  console.log("Change type of event: " + event.type);
+}
 ```
 
 ## See also
 
-- [Share data and events between Excel custom functions and task pane tutorial](../tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md)
\ No newline at end of file
+- [Configure your Office Add-in to use a shared JavaScript runtime](configure-your-add-in-to-use-a-shared-runtime.md)
+- [Share data and events between Excel custom functions and task pane tutorial](../tutorials/share-data-and-events-between-custom-functions-and-the-task-pane-tutorial.md)
```

Comment:
Would it be relevant to include a link to [Work with Events using the Excel JavaScript API](excel-add-ins-events.md) here? This page: https://docs.microsoft.com/en-us/office/dev/add-ins/excel/excel-add-ins-events

Reply:
added!